### ISSUE TITLE
Published version on npm doesn't generate bin directory.

### ISSUE THREAD

**Issue by joshfarrant** [Nickersoft/push.js#10]
The version published on npm doesn't include or generate the bin (now named dist) directory. Might be worth publishing a new version with the most recent changes.

**Comment by KuTi** [comment]
Hi,
I've fixed it by

- Open **package.json**

Find:
```
  "main": "bin/push.js",
```
Replace as:
```
  "main": "push.js",
```

- Open **push.js**

find:
```
module.exports = (factory(global, global.document))();
```
replace as:
```
module.exports = new (factory(global, global.document))();
```

**Comment by joshfarrant** [comment]
A good temporary workaround, however this could do with being fixed on the published version too.

**Comment by Nickersoft** [comment]
This was fixed in PR #8. The npm package now only contains a package.json and a `dist` directory. Version 0.0.3 was published this morning containing the fix.
